### PR TITLE
Fix vm-cirros-cluster-csmall VirtualMachineClusterInstancetype name value.

### PR DESCRIPTION
**Pull request by jcanocan**
**What this PR does / why we need it**:

Change VirtualMachineClusterInstancetype name value from "csmall" to "cluster-csmall" for making it work with the ClusterInstanceType defined at cluster-csmall.yaml.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: examples/vm-cirros-cluster-csmall.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   instancetype:
     kind: VirtualMachineClusterInstancetype
-    name: csmall
+    name: cluster-csmall
```
Comment (lyarwood) [inline examples/vm-cirros-cluster-csmall.yaml:11]: You actually need to update the following code and then run `make generate` that in turn generates the examples :

https://github.com/akrejcir/kubevirt/blob/cd0a28c2c34f5e447ff2d3465ca5eb2708feae66/tools/vms-generator/utils/utils.go#L1261-L1264

More details below:

https://github.com/kubevirt/kubevirt/blob/main/docs/getting-started.md#automatic-code-generation

**Review by lyarwood — CHANGES_REQUESTED** [review]

File: examples/vm-cirros-cluster-csmall.yaml (hunk 1 above)
Comment (jcanocan) [reply examples/vm-cirros-cluster-csmall.yaml:11]: Done. Hope everything is fine now.

Comment (lyarwood) [reply examples/vm-cirros-cluster-csmall.yaml:11]: ACK thanks, I'll just let `make generate` run in CI before giving my `/lgtm`

File: tools/vms-generator/utils/utils.go [hunk 2]
```diff
@@ -1259,7 +1259,7 @@ func GetVmCirrosClusterInstancetypeComputeSmall() *v1.VirtualMachine {
 	})
```
Comment (lyarwood) [inline tools/vms-generator/utils/utils.go:1259]: Can you update `VmCirrosInstancetypeComputeSmall` to `VmCirrosClusterInstancetypeComputeSmall` as well?

Comment (jcanocan) [reply tools/vms-generator/utils/utils.go:1259]: Done. Please, check if it is correct.

**Review by lyarwood — CHANGES_REQUESTED** [review]
You just need to run `make generate` again to apply your changes to the examples and then this should correct.

```
$ gh pr checkout 8342
[..]
$ make generate-verify
[..]
$ git diff
diff --git a/examples/vm-cirros-cluster-csmall.yaml b/examples/vm-cirros-cluster-csmall.yaml
index efc4bb9e7..584a7eea2 100644
--- a/examples/vm-cirros-cluster-csmall.yaml
+++ b/examples/vm-cirros-cluster-csmall.yaml
@@ -3,8 +3,8 @@ apiVersion: kubevirt.io/v1
 kind: VirtualMachine
 metadata:
   labels:
-    kubevirt.io/vm: vm-cirros-csmall
-  name: vm-cirros-csmall
+    kubevirt.io/vm: vm-cirros-cluster-csmall
+  name: vm-cirros-cluster-csmall
 spec:
   instancetype:
     kind: VirtualMachineClusterInstancetype
@@ -13,7 +13,7 @@ spec:
   template:
     metadata:
       labels:
-        kubevirt.io/vm: vm-cirros-csmall
+        kubevirt.io/vm: vm-cirros-cluster-csmall
     spec:
       domain:
         devices:
[..]
$ ./cluster-up/kubectl.sh apply -f examples/cluster-csmall.yaml -f examples/vm-cirros-cluster-csmall.yaml 
selecting docker as container runtime
virtualmachineclusterinstancetype.instancetype.kubevirt.io/cluster-csmall created
virtualmachine.kubevirt.io/vm-cirros-cluster-csmall created
$  ./cluster-up/virtctl.sh start vm-cirros-cluster-csmall
selecting docker as container runtime
VM vm-cirros-cluster-csmall was scheduled to start
$ ./cluster-up/kubectl.sh get vms
selecting docker as container runtime
NAME                       AGE     STATUS    READY
vm-cirros-cluster-csmall   42s     Running   True

```

**Comment by jcanocan** [comment]
> You just need to run `make generate` again to apply your changes to the examples and then this should be correct.
> 
> ```
> $ gh pr checkout 8342
> [..]
> $ make generate-verify
> [..]
> $ git diff
> diff --git a/examples/vm-cirros-cluster-csmall.yaml b/examples/vm-cirros-cluster-csmall.yaml
> index efc4bb9e7..584a7eea2 100644
> --- a/examples/vm-cirros-cluster-csmall.yaml
> +++ b/examples/vm-cirros-cluster-csmall.yaml
> @@ -3,8 +3,8 @@ apiVersion: kubevirt.io/v1
>  kind: VirtualMachine
>  metadata:
>    labels:
> -    kubevirt.io/vm: vm-cirros-csmall
> -  name: vm-cirros-csmall
> +    kubevirt.io/vm: vm-cirros-cluster-csmall
> +  name: vm-cirros-cluster-csmall
>  spec:
>    instancetype:
>      kind: VirtualMachineClusterInstancetype
> @@ -13,7 +13,7 @@ spec:
>    template:
>      metadata:
>        labels:
> -        kubevirt.io/vm: vm-cirros-csmall
> +        kubevirt.io/vm: vm-cirros-cluster-csmall
>      spec:
>        domain:
>          devices:
> [..]
> $ ./cluster-up/kubectl.sh apply -f examples/cluster-csmall.yaml -f examples/vm-cirros-cluster-csmall.yaml 
> selecting docker as container runtime
> virtualmachineclusterinstancetype.instancetype.kubevirt.io/cluster-csmall created
> virtualmachine.kubevirt.io/vm-cirros-cluster-csmall created
> $  ./cluster-up/virtctl.sh start vm-cirros-cluster-csmall
> selecting docker as container runtime
> VM vm-cirros-cluster-csmall was scheduled to start
> $ ./cluster-up/kubectl.sh get vms
> selecting docker as container runtime
> NAME                       AGE     STATUS    READY
> vm-cirros-cluster-csmall   42s     Running   True
> ```

Got it!! Thanks.

**Comment by lyarwood** [comment]
@jcanocan when you get a chance can you run `make generate` and push your branch?

**Comment by jcanocan** [comment]
> @jcanocan when you get a chance can you run `make generate` and push your branch?

Yes, done. Sorry about the delay.

**Comment by jcanocan** [comment]
@davidvossel Could you please take a look a this when you have availability? Thanks a lot.

**Review by xpivarc — COMMENTED** [review]
/approve

**Pull request merged**